server/monitor/src/main/resources/templates/index.ftl

```diff
@@ -0,0 +1,72 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<html>
+  <head>
+    <title>${title} - Accumulo ${version}</title>
+    <meta http-equiv="Content-Type" content="test/html" />
+    <meta http-equiv="Content-Script-Type" content="text/javascript" />
+    <meta http-equiv="Content-Style-Type" content="text/css" />
+    <link rel="shortcut icon" type="image/jng" href="/resources/favicon.png" />
+    <script src="/resources/global.js" type="text/javascript"></script>
+    <script src="/resources/functions.js" type="text/javascript"></script>
+    
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+    <script language="javascript" type="text/javascript" src="/resources/flot/jquery.flot.js"></script>
+    <script language="javascript" type="text/javascript" src="/resources/flot/jquery.flot.time.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/css/select2.min.css" rel="stylesheet" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/js/select2.min.js"></script>
+    
+    <link rel="stylesheet" type="text/css" href="/resources/screen.css" media="screen" />
+    <script>
+      /**
+       * Sets up autorefresh on initial load
+       */
+      $(document).ready(function() {
+        setupAutoRefresh();
+      });
+    </script>
+    <#if js??>
+      <script src="/resources/${js}"></script>
+    </#if>
+    <script src="/resources/sidebar.js"></script>
+  </head>
+
+  <body>
+    <#include "/templates/modals.ftl">
+    <div id="content-wrapper">
+      <div id="content">
+        <div id="sidebar" class="navbar navbar-inverse navbar-fixed-top">
+          <!--<#include "/templates/header.ftl">-->
+          <#include "/templates/sidebar.ftl">
```

Comment:
Thanks, initially it was a sidebar that turned into a navbar and I forgot to rename it. Yes, the comment and the file can be removed. Will do this tomorrow.